src/Sfa.Das.ApprenticeshipInfoService.Infrastructure/Elasticsearch/StandardRepository.cs

```diff
@@ -78,6 +80,44 @@ public Standard GetStandardById(string id)
             return response;
         }
 
+        public List<Standard> GetStandardsById(List<int> ids, int page)
+        {
+            var skip = GetSkipAmount(page);
+	        var elements = ids.Skip(skip).Take(Take);
+            var results = _elasticsearchCustomClient.Search<StandardSearchResultsItem>(
+                s =>
+                    s.Index(_applicationSettings.ApprenticeshipIndexAlias)
+                        .Type(Types.Parse("standarddocument"))
+                        .Take(Take)
+                        .Query(q => q
+                            .Terms(t => t
+                                .Field(fi => fi.StandardId).Terms(elements))));
+
```

Comment:
Do we need to add some ordering here? Just to make sure that we reliably return consistent data when paging? or are we relying on them replaying the ids each time in the same order?

Reply:
@dashton82 fixed